tree/treedb.go

```diff
@@ -182,6 +185,30 @@ func SetNamespace(
 	return context.TreeDB
 }
 
+const (
+	treeRoutinePoolSize = 10240
```

Comment:
Would this be too large?

Reply:
updated, and I made it configurable, default changed to `1024`